db/migrate/20190422154904_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string   :first_name
+      t.string   :last_name
+      t.string   :email
+      t.string   :password
+      t.string   :password_confirmation
+      t.integer  :role
+      t.integer  :organization_id
```

Comment:
add index by organization_id in ALL models where you have it
think about indexes

Reply:
use `t.belongs_to :organization` to create association and all necessary indexes and columns